docs/howto/operate/manual-deploy.md

```diff
@@ -1,52 +1,74 @@
-# Manually deploy a hub
+# Set up and use the the deployment scripts locally
 
-The [`deploy.py`](https://github.com/2i2c-org/pilot-hubs/blob/master/deploy.py) script is a cmd utility built to help deploy one or all the hubs to their Kubernetes clusters, but  also to build the user image. Checkout the [](../configure/update-env.md) section on how to use the deploy script to update the user environment.
+While deploys generally go through our GitHub Actions workflow, sometimes you
+need to deploy from your laptop - primarily when testing changes on staging or
+actively fixing an outage. This isn't ideal, but this is where we are now.
 
-## Deploy script arguments and usage
-Running:
-```
-python3 deploy.py deploy --help
-```
-Will provide information about the `deploy.py` deploy options:
+Our deployment scripts live in the [`deploy/`](https://github.com/2i2c-org/pilot-hubs/blob/master/deploy.py)
+of this repository, and can deploy one or more hubs to our clusters. 
 
-```bash
-usage: deploy.py deploy [-h] [--skip-hub-health-test]
-                        [cluster_name] [hub_name]
 
-positional arguments:
-  cluster_name
-  hub_name
+## Setting up local environment
 
-optional arguments:
-  -h, --help            show this help message and exit
-  --skip-hub-health-test
+1. Create a virtual environment for use with this repository
 
-```
+   ```bash
+   python3 -m venv .
+   source bin/activate
+   ```
 
-* `cluster_name` argument
+   You can also use `conda` if you prefer instead.
 
-    Deploys all the hubs in the cluster passed as argument. Cluster config is read
-    from `config/hubs/{cluster_name}.cluster.yaml`
-    ```bash
-    python3 deploy.py deploy <cluster>
-    ```
-* `hub_name` argument
+2. Install python packages required by our deployment script
+   
+   ```bash
+   pip install -r requirements.txt
+   ```
 
-    Deploys only the hub passed as argument. 
+3. Install the [Google Cloud SDK](https://cloud.google.com/sdk) so
```

Comment:
What about for hubs that are on AWS or Azure? If those aren't supported right now, we should add a `note:` block to make it clear

Reply:
I've updated the text here to clarify.